pulp_rpm/app/modulemd.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+from pulp_rpm.app.constants import PULP_MODULE_ATTR, MODULEMD_MODULE_ATTR
+from pulp_rpm.app.models import Modulemd
+from pulp_rpm.app.serializers import ModulemdSerializer
+
+import gi
+gi.require_version('Modulemd', '2.0')
+from gi.repository import Modulemd as mmdlib
```

Comment:
from gi.repository import Modulemd as mmdlib  #noqa

Reply:
done, thank you